backend/src/test/java/ch/puzzle/pcts/mapper/CalculationMapperTest.java

```diff
@@ -68,11 +84,30 @@ private Role createRole() {
     }
 
     private Calculation createCalculation(Member member, Role role) {
-        return new Calculation(CALC_ID, member, role, STATE, PUBLICATION_DATE, PUBLICIZED_BY);
+        Calculation calculation = Calculation.Builder
+                .builder()
+                .withId(CALC_ID)
+                .withMember(member)
+                .withRole(role)
+                .withState(STATE)
+                .withPublicationDate(PUBLICATION_DATE)
+                .withPublicizedBy(PUBLICIZED_BY)
+                .withDegrees(List.of())
+                .withExperiences(List.of())
+                .withCertificates(List.of())
+                .build();
+        calculation.setPoints(POINTS);
+        return calculation;
     }
 
     private CalculationInputDto createInput() {
-        return new CalculationInputDto(MEMBER_ID, ROLE_ID, STATE);
+        return new CalculationInputDto(MEMBER_ID,
+                                       STATE,
+                                       ROLE_ID,
+                                       List.of(1L, 2L),
+                                       List.of(10L),
+                                       List.of(mock(DegreeCalculationInputDto.class)),
```

Comment:
Maybe think about extracting into a var in order to be able to actual mock the specific functions to prevent a NPE exception